source/renderer/app/containers/settings/Settings.js

```diff
@@ -27,10 +27,14 @@ export default class Settings extends Component<InjectedContainerProps> {
 
   render() {
     const { isFlight } = global;
-    const { actions, children } = this.props;
+    const { actions, stores, children } = this.props;
+    const { isSynced } = stores.networkStatus;
```

Comment:
A minor thing, can you extract networkStatus and app from stores into separate consts, it adds more readability.

`const { app, networkStatus } = stores;
`
Then reuse it bellow 

`const { isSynced } = networkStatus;`

`const { currentRoute } = app;`

Reply:
Done